apps/meteor/tests/e2e/admin-feature-preview.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { Users } from './fixtures/userStates';
+import { test, expect } from './utils/test';
+
+test.use({ storageState: Users.admin.state });
+
+test.describe.serial('admin-feature-preview', () => {
```

Comment:
each `test(...)` opens a new browser tab which takes time, are you sure this is really necessary? you dont need to create one test each assertion. maybe `test.step(...)`